sources/packages/api/src-sql/StudentAppeals/Create-student-appeals.sql

```diff
@@ -0,0 +1,28 @@
+CREATE TABLE IF NOT EXISTS sims.student_appeals (
```

Comment:
Asking just for my understanding. Why have we kept student_appeals and student_appeal_requests in a different table?

Reply:
The appeal table is to group the many possible requests. The many appeal requests must be submitted at once and assessed by the Ministry together. The appeals table is the table actually related to the assessment table. So one assessment can be related to one or many approved student requests.